Sources/Swift/SwiftDescriptor.swift

```diff
@@ -8,4 +8,9 @@ public class SwiftDescriptor: NSObject {
         return String(describing: type(of: object))
     }
     
+    @objc
+    public static func getSwiftErrorDescription(_ error: Error) -> String? {
+        return String(describing: error)
+    }
```

Comment:
I guess you missed my update in the description above 

Update: No need to worry about PII in a major as we already would send PII of Errors with https://github.com/getsentry/sentry-cocoa/blob/9d61bea77651d44e030326c34b2224d4076a9184/Sources/Sentry/SentryClient.m#L265-L267

Reply:
> I guess you missed my update in the description above

Yeap I did. I would like to suggest to only update description if something is wrong, change of direction in the PR should be describe in a new comment. Or both, since GH itself don't warn about description changing.

Thanks for clarifying!